go.sum

```diff
@@ -37,33 +37,20 @@ github.com/ActiveState/archiver v3.1.1+incompatible h1:uSvaGvIm9/QszTShi5ojEfgxZ
 github.com/ActiveState/archiver v3.1.1+incompatible/go.mod h1:tiVVhFlOSJ4y8iEjflUccKQk0xV5f2CuQwaAom2h3rg=
 github.com/ActiveState/go-colortext v0.0.0-20190402165444-9cd201a66eb0 h1:j6rNAf34TpJO0jTXRiiaOv94cveQ+mh7grzjLEmfwAY=
 github.com/ActiveState/go-colortext v0.0.0-20190402165444-9cd201a66eb0/go.mod h1:Mjj0kASzEmqC32vpw3ZxZ3KS6qjLjZd8L/zVN1R6pVo=
-github.com/ActiveState/go-conpty v0.3.0 h1:AuYWJvChpwq/gavbFDshFSSciTkMJb1Plx+iRPrnMME=
-github.com/ActiveState/go-conpty v0.3.0/go.mod h1:ju8+TDvkEjrFhDKg5FnMg7odjlYCcl6X8Sq3ZEqMbZ8=
-github.com/ActiveState/go-expect v0.2.0/go.mod h1:2iws19tf0uTZb4FMPDTWC28pO6qYkVJzHoYx4dPxfwM=
```

Comment:
It removes the private packages here.  Last time I updated, I must have forgotten to run `go mod vendor` so the actually never ended up in the vendor directories.